modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -19,6 +19,7 @@
         :current="docFields.data"
         :published="published"
         :show-edit="false"
+        :can-delete-draft="moduleOptions.canDeleteDraft"
```

Comment:
passing this info to AposDocEditor context menu (was already working for manager modal items)